BasicLayoutsCodelab/app/src/main/java/com/codelab/basiclayouts/MainActivity.kt

```diff
@@ -221,45 +227,52 @@ fun HomeScreen(modifier: Modifier = Modifier) {
     ) {
         Spacer(Modifier.height(16.dp))
         SearchBar(Modifier.padding(horizontal = 16.dp))
+        Spacer(Modifier.height(32.dp))
```

Comment:
It looks like before the HomeSection composable made sure there was enough spacing at its top. Is there a reason you are doing this with a Spacer now?

Reply:
done

Reply:
the initial spacing doesn't align with the figma file